concent_api/verifier/utils.py

```diff
@@ -354,3 +354,61 @@ def download_archives_from_storage(file_transfer_token, subtask_id, package_path
                 ErrorCode.VERIFIER_FILE_DOWNLOAD_FAILED,
                 subtask_id=subtask_id,
             )
+
+
+def parse_result_files_with_frames(frames, result_files_list, output_format):
+    frames_to_result_files_map = {}
+    for frame_number in frames:
+        for result_file_name in result_files_list:
+            if re.search(f'_{frame_number:>04}.{output_format.lower()}$', result_file_name) is not None:
```

Comment:
You can combine this into single condition.

Reply:
Done